src/fshtypes/Logical.ts

```diff
@@ -22,3 +24,31 @@ export class Logical extends FshStructure {
     return `${metadataFSH}${rulesFSH.length ? EOL + rulesFSH : ''}`;
   }
 }
+
+// see https://hl7.org/fhir/extensions/ValueSet-type-characteristics-code.html
+export type LogicalCharacteristic =
+  | 'has-target'
+  | 'has-range'
+  | 'is-continuous'
+  | 'has-length'
+  | 'has-size'
+  | 'can-bind'
+  | 'has-units'
+  | 'do-translations'
+  | 'can-be-target';
+
+export function isLogicalCharacteristic(s: string): s is LogicalCharacteristic {
+  return (
+    [
+      'has-target',
+      'has-range',
+      'is-continuous',
+      'has-length',
+      'has-size',
+      'can-bind',
+      'has-units',
+      'do-translations',
+      'can-be-target'
+    ].indexOf(s) >= 0
+  );
+}
```

Comment:
Hard-coding these codes here is a little brittle because as soon as the next version (or pre-release) of FHIR comes out, the list could change. We look up the code system in the exporter so that we can avoid hard-coding the valid codes there; but I think we lose the value of that if we're going to validate against a hard-coded list in the importer anyway. Do we need to validate in the importer _and_ exporter?